services/slam/builtin/builtin_test.go

```diff
@@ -331,28 +390,12 @@ func TestGeneralNew(t *testing.T) {
 
 		// Create slam service
 		logger := golog.NewTestLogger(t)
-		_, err := createSLAMService(t, attrCfg, logger, false)
+		_, err := createSLAMService(t, attrCfg, logger, false, false)
 		test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error with slam service slam process:")
 
 		delete(slam.SLAMLibraries, "test")
 	})
 
-	t.Run("New slam service the fails at slam process due to binary location", func(t *testing.T) {
```

Comment:
I removed this test because it duplicates this one: https://github.com/viamrobotics/rdk/blob/2a05492446addf71bbfc278fb9e4fdb1545c433e/services/slam/builtin/builtin_test.go#L839. Once the orbslam binary is in place, this test will start failing, since it expects the orbslam binary to be missing.